mindmeld/gazetteer.py

```diff
@@ -125,7 +126,9 @@ def _update_entity(self, entity, popularity, keep_max=True):
         """
         # Only update the relevant data structures when the entity isn't
         # already in the gazetteer. Update the popularity either way.
-        tokenized_gaz_entry = tuple(token["entity"] for token in self.tokenizer.tokenize(entity))
+        tokenized_gaz_entry = tuple(
+            token["text"] for token in self.text_preparation_pipeline.tokenize(entity)
```

Comment:
why text instead of entity?

Reply:
Good point, I'll switch this to "entity" and "tokenize_and_normalize" as that is more consistent with the old "tokenize"